### PR TITLE
Feat routes

### PR DESCRIPTION
**Pull request by sai-tanush**

### CODE REVIEW

File: controllers/Auth.js [hunk 1]
```diff
@@ -24,31 +25,28 @@ exports.sendOtp = async (req, res) => {
     }
 
     //generate OTP
-    let otp = otpGenerator.generate(6, {
+    var otp = otpGenerator.generate(6, {
```
Comment (pavankumar-v) [inline controllers/Auth.js]: what! why broo?!! `var`

File: controllers/Profile.js [hunk 2]
```diff
@@ -16,6 +17,7 @@ exports.updateProfile = async (req, res) => {
 		profile.dateOfBirth = dateOfBirth;
 		profile.about = about;
 		profile.contactNumber = contactNumber;
+    profile.gender = gender;
```
Comment (pavankumar-v) [inline controllers/Profile.js:20]: why are you not streamlining code formatting

**Review by pavankumar-v — CHANGES_REQUESTED** [review]

File: controllers/Auth.js (hunk 1 above)
Comment (sai-tanush) [reply controllers/Auth.js]: my bad

File: controllers/Profile.js (hunk 2 above)
Comment (sai-tanush) [reply controllers/Profile.js:20]: i forgot to setup in backend

File: controllers/Auth.js (hunk 1 above)
Comment (sai-tanush) [reply controllers/Auth.js]: resolved

**Review by pavankumar-v — APPROVED** [review]

**Pull request merged**